umh.docs.umh.app/content/en/docs/datacontracts/customdatacontracts/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Custom Data Contracts"
+menuTitle: "Custom Data Contracts"
+description: "In addition to the standard data contracts provided, you can add your own."
+weight: 2000
+draft: true
```

Comment:
Maybe still add a couple of words here, people want to use it. See also https://discord.com/channels/700613971941785680/984083100520808478/1313876222429630525

Can provide more details in DM if necessary

Reply:
Can you specify, what is missing in your opinion?